source/blind_jump/state/titleScreenState.cpp

```diff
@@ -154,18 +154,21 @@ TitleScreenState::update(Platform& pfrm, Game& game, Microseconds delta)
                 const u8 right_x = text.coord().x + width + 1;
                 return {left_x, right_x};
             };
+
```

Comment:
The "new game" text is longer in italian, so I made a minor adjustment to the cursor flicker animation on the title screen

Reply:
I tried to keep the same amounts of characters but it was possible everywhere...
We have long words 😂